schema/cat-vrs/cat-vrs-source.yaml

```diff
@@ -42,6 +42,14 @@ $defs:
         ordered: false
         items:
           $ref: "#/$defs/Constraint"
+      mappings:
```

Comment:
I'm think we should reevaluate whether it makes sense to keep `mappings` in `CategoricalVariant`.